projects/focil-geth-and-prysm-poc.md

```diff
@@ -0,0 +1,80 @@
+# Fork-Choice Enforced Inclusion Lists (FOCIL) Proof of Concept in Geth and Prysm
+
+Enhancing Decentralization and Censorship Resistance in Block Production
+
+## Motivation
+
+**Centralization Issue:**
+- Block building rights are auctioned to specialized entities known as builders.
+- Over the past year, a few sophisticated builders have dominated block production due to economies of scale.
+
+**Censorship Resistance:**
+- Concentrated block production weakens the network’s censorship resistance properties.
+- 60% of builders filter transactions from sanctioned addresses, while 90% of validators do not engage in censorship.
+
+## Project Description
+
+A randomly selected set of validators forms an inclusion list ($IL$) committee to create and broadcast local inclusion lists. The block producer aggregates these lists into a canonical aggregate ($IL_{agg}$), which is included in the block and validated by attesters to ensure block validity.
```

Comment:
A bit more context would be nice here, maybe just adding the link to the research post.

Reply:
sure.